test/common/config/new_grpc_mux_impl_test.cc

```diff
@@ -328,7 +328,8 @@ TEST_F(NewGrpcMuxImplTest, ConfigUpdateWithAliases) {
   response->mutable_resources()->at(0).add_aliases("prefix/domain1.test");
   response->mutable_resources()->at(0).add_aliases("prefix/domain2.test");
 
-  grpc_mux_->onDiscoveryResponse(std::move(response), control_plane_stats_);
+  EXPECT_LOG_CONTAINS("debug", "for " + type_url + " from HAL 9000",
```

Comment:
No change to the `grpc_mux_impl_test.cc` as well?

Reply:
I was lazy, and wasn't sure how strict we were about testing for logging. Added.